src/converter/FlameGraph.java

```diff
@@ -55,6 +57,8 @@ public FlameGraph(String... args) {
                 minwidth = Double.parseDouble(args[++i]);
             } else if (arg.equals("--skip")) {
                 skip = Integer.parseInt(args[++i]);
+            } else if (arg.equals("--highlightstacks")) {
+                highlightStacksPattern = Pattern.compile(".*" + args[++i] + ".*");
```

Comment:
Where is `highlightstacks` field initialized, if no argument specified on the command line? Seems like there will be a NullPointerException in such case.